athenz/data_source_role.go

```diff
@@ -29,6 +29,12 @@ func DataSourceRole() *schema.Resource {
 				Elem:     &schema.Schema{Type: schema.TypeString},
 				Set:      schema.HashString,
 			},
+			"trust": {
+				Type:        schema.TypeString,
+				Description: "The domain, which this role is trusted to",
```

Comment:
It should be fully qualified role name (<domain>.<role>)

Reply:
What should be? I don't understand this comment - delegated roles trust another domain; not another role - the mechanism on the client domain end is for _their_ role to "assume_role" the delegated role in this domain.

Reply:
Indeed.
I've confused.
`(oath_tools) dguttman@F265P9343M calypso-ui (sd_aws) $ zms-cli help add-delegated-role
 syntax:
   -d domain add-delegated-role role trusted_domain
 parameters:
   domain       : name of the domain that role belongs to
   role         : name of the cross-domain/trust delegated role
   trust_domain : name of the cross/trusted domain name
 examples:
   -d coretech add-delegated-role tenant.sports.readers sports`